pkg/node/node.go

```diff
@@ -286,6 +287,21 @@ func (n Node) TaintSpotItn(nodeName string, eventID string) error {
 	return addTaint(k8sNode, n, SpotInterruptionTaint, eventID, corev1.TaintEffectNoSchedule)
 }
 
+// LogPods logs all the pod names on a node
+func (n Node) LogPods(nodeName string) error {
+	podList, err := n.fetchAllPods(nodeName)
+	if err != nil {
+		return fmt.Errorf("Unable to fetch all pods from API: %w", err)
+	}
+	podNamesArr := zerolog.Arr()
+	for _, pod := range podList.Items {
+		podNamesArr = podNamesArr.Str(pod.Name)
+	}
+	log.Log().Array("pod_names", podNamesArr).Msgf("Pods on node %s", nodeName)
```

Comment:
Can you log the node name with `Str("node_name", nodeName)` instead?

Reply:
Sure. I have made the change.